tools/schemacode/bidsschematools/render.py

```diff
@@ -287,9 +287,31 @@ def make_filename_template(schema, n_dupes_to_combine=6, **kwargs):
     paragraph = paragraph.rstrip()
     codeblock = "Template:\n```Text\n" + paragraph + "\n```"
     codeblock = codeblock.expandtabs(4)
+    codeblock = append_filename_template_legend(codeblock)
     return codeblock
 
 
+def append_filename_template_legend(text=None):
+
+    if text is None:
+        return
+
+    text += """\n\n
+    **Legend:**
+    \n
+    - Filename entities or folders between square brackets
+      (for example `[_ses-<label>]`) are OPTIONAL.
+    - See the glossary for a definition of
+      [`label`](../99-appendices/14-glossary.md#label)
+      and [`index`](../99-appendices/14-glossary.md#index).
+    - See the [entity page](../99-appendices/09-entities.md)
+      for a definition of each entity.
+    \n
+    """
```

Comment:
For some reason, this is rendered as a code block, so the Markdown elements aren't showing up.

![image](https://user-images.githubusercontent.com/8228902/185217912-2f43b898-b61a-4d7d-829e-94a53a4b3154.png)

Reply:
It's tabbed out.
```suggestion
    text += """

**Legend:**

- Filename entities or folders between square brackets (for example `[_ses-<label>]`) are OPTIONAL.
- See the glossary for a definition of [`label`](../99-appendices/14-glossary.md#label)
  and [`index`](../99-appendices/14-glossary.md#index).
- See the [entity page](../99-appendices/09-entities.md) for a definition of each entity.

"""
```